scripts/download_backtest_data.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+"""This script generate json data from bittrex"""
+import json
+import sys
+import os
+
+from freqtrade import (exchange, arguments, misc)
+
+DEFAULT_DL_PATH = 'freqtrade/tests/testdata'
+
+arguments = arguments.Arguments(sys.argv[1:], 'download utility')
+arguments.testdata_dl_options()
+args = arguments.parse_args()
+
+TICKER_INTERVALS = ['1m', '5m']
+PAIRS = []
+
+if args.pairs_file:
+    with open(args.pairs_file) as file:
+        PAIRS = json.load(file)
+PAIRS = list(set(PAIRS))
+
+dl_path = DEFAULT_DL_PATH
+if args.export and os.path.exists(args.export):
+    dl_path = args.export
+
+print(f'About to download pairs: {PAIRS} to {dl_path}')
+
+# Init Bittrex exchange
+exchange._API = exchange.init_ccxt({'key': '',
+                                    'secret': '',
+                                    'name': 'bittrex'})
+
+for pair in PAIRS:
+    for tick_interval in TICKER_INTERVALS:
+        print(f'downloading pair {pair}, interval {tick_interval}')
+        data = exchange.get_ticker_history(pair, tick_interval)
```

Comment:
I think looping over time to get a longer history should be added.
Binace only gives you a 500 candles (max 1000) per request - and the earliest i could get was 2017-10-04 (but better than only the last 2 days).


the following sample/mockup code (i ran this in a jupyter notebook) gives me more history - but only when looping with "shift".
``` python
date = arrow.now().shift(days=-20)
ct.fetch_ohlcv(symbol, timeframe='5m', limit=1000, since=int(date.datetime.timestamp()) * 1000)
```

Now this is probably not the place to implement this, but rather `get_ticker_history` is ... but i belive it should be implemented to allow proper backtesting (over longer periods of time).